tests/serializers/test_data.py

```diff
@@ -392,9 +392,13 @@ def serializerTest(format, self):
 
     # Create all the objects defined in the test data
     objects = []
+    tables = set()
     instance_count = {}
-    for (func, pk, klass, datum) in test_data:
-        with connection.constraint_checks_disabled():
```

Comment:
Note that changing the placement of this call outside the loop drops the call count for constraint_checks_disabled() by roughly 500. In other words, this place is responsible for 2/3 of all calls.